velox/common/caching/AsyncDataCache.cpp

```diff
@@ -153,6 +164,9 @@ CachePin CacheShard::findOrCreate(
       } else {
         ++numHit_;
       }
+      if (found->size() != size) {
+        LOG(INFO) << "Different size reqd than found";
```

Comment:
What does this log message mean? Why it is OK if sizes don't match?

typo: reqd -> required + would be nice to add the two values that do not match